mongo_repository.go

```diff
@@ -304,3 +323,35 @@ func fromRecords(records []record) []Event {
 
 	return events
 }
+func MigrateMongoDB(mongoDB *mongo.Database, dir string) error {
```

Comment:
```suggestion
}

func MigrateMongoDB(mongoDB *mongo.Database, dir string) error {
```

Si tu mets `gofumpt: true` dans la conf de gopls t'auras ce genre de fixes en plus (c'est un superset de gofmt).